gcp/rakefiles/test.rake

```diff
@@ -0,0 +1,23 @@
+desc '[TEST] Run Locust swarm against Preferences service in current cluster'
+task :test_preferences => [:set_vars] do
+  sh "#{@exekube_cmd} rake xk[' \
+    xk down live/dev/locust && \
+    sleep 30 && \
```

Comment:
Can you add `echo Sleeping 30s...` here so the user doesn't think the process is stuck?

Reply:
See next tread.